<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connessione</class>
 <widget class="QDialog" name="connessione">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connessione al database Lyibrary</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../lylibrary/biblio.qrc">
    <normaloff>:/images/svn-commit.png</normaloff>:/images/svn-commit.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Crea database locale</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>Processo in corso</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="p_crea">
          <property name="text">
           <string>Crea DB</string>
          </property>
          <property name="icon">
           <iconset resource="../lylibrary/biblio.qrc">
            <normaloff>:/images/vcs_add.png</normaloff>:/images/vcs_add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Enter</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QTextEdit" name="textEdit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(195, 201, 77);</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>218</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>120</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../lylibrary/biblio.qrc">:/images/backup_restor.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>Creazione del Database Lylibrary per la prima installazione</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Password mysql</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="password"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="add_mysql_socket">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../lylibrary/biblio.qrc">
       <normaloff>:/images/go-next.png</normaloff>:/images/go-next.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Connetti database in rete</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Indirizzo di rete</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ip_dat"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Porta</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="port">
          <property name="maximum">
           <number>20000</number>
          </property>
          <property name="value">
           <number>3306</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Nome database in rete</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="n_dat_lan"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Nome utente</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="user"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="pwd_lan">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>272</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QPushButton" name="open_db">
          <property name="text">
           <string>Connetti</string>
          </property>
          <property name="icon">
           <iconset resource="../lylibrary/biblio.qrc">
            <normaloff>:/images/system-run.png</normaloff>:/images/system-run.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>134</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="esci">
   <property name="icon">
    <iconset resource="../lylibrary/biblio.qrc">
     <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
   </property>
   <property name="text">
    <string>Esci</string>
   </property>
  </action>
  <action name="action_crea">
   <property name="icon">
    <iconset resource="../lylibrary/biblio.qrc">
     <normaloff>:/images/vcs_add.png</normaloff>:/images/vcs_add.png</iconset>
   </property>
   <property name="text">
    <string>Crea database</string>
   </property>
  </action>
  <action name="action_elimina">
   <property name="icon">
    <iconset resource="../lylibrary/biblio.qrc">
     <normaloff>:/images/vcs_remove.png</normaloff>:/images/vcs_remove.png</iconset>
   </property>
   <property name="text">
    <string>Elimina database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>password</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>add_mysql_socket</tabstop>
  <tabstop>ip_dat</tabstop>
  <tabstop>port</tabstop>
  <tabstop>n_dat_lan</tabstop>
  <tabstop>user</tabstop>
  <tabstop>pwd_lan</tabstop>
  <tabstop>open_db</tabstop>
  <tabstop>p_crea</tabstop>
 </tabstops>
 <resources>
  <include location="../lylibrary/biblio.qrc"/>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
